apps/src/p5lab/AnimationTab/AnimationList.jsx

```diff
@@ -20,23 +20,39 @@ class AnimationList extends React.Component {
     onNewItemClick: PropTypes.func.isRequired,
     spriteLab: PropTypes.bool.isRequired,
     hideBackgrounds: PropTypes.bool.isRequired,
+    hideCostumes: PropTypes.bool.isRequired,
     labType: PropTypes.string.isRequired
   };
 
   render() {
+    let buttonLabelNewAnimation;
+    if (this.props.spriteLab) {
```

Comment:
nit: we have a convention we often use to make accessing a bunch of props/state in the same function more readable:

```javascript
render() {
  const {spriteLab, onNewItemClick, hideCostumes} = this.props;

  // use local variables throughout the function to improve readability
}
```

i know you're following the existing code in this case, but i figured i'd call it out either way in case you didn't know this was an explicit convention we use!

Reply:
Thanks for showing me this convention. I changed this across a few of the animation tab components, but I'm seeing dozens of other cases just like it throughout the Animation Picker components, etc. Would you recommend cleaning these all up?

Also as a clarification, do we use this when a bunch of props are used even if they are only used once each within the function, or do we primarily use it when the same prop is used multiple times within a function?

Reply:
> Would you recommend cleaning these all up?

no, not necessarily. i usually only do it if i'm making considerable changes in the function and this cleanup would improve readability.

> do we use this when a bunch of props are used even if they are only used once each within the function, or do we primarily use it when the same prop is used multiple times within a function?

this is a great question! i think this is where the convention becomes more subjective 😄 if the function is really simple and the prop is only used once, it's probably unnecessary to save the prop to a local variable. but if the function is really complex and/or the prop is used more than once, this convention really helps improve readability